tests/python/test_flash_attn.py

```diff
@@ -90,35 +90,37 @@ def run_flash_attention(self, m, n, k, p, kTM, kTN, kTK, kTP):
         V = torch.randn(n, p, device='cpu')
         O = torch.empty(m, p, device='cpu')
 
-        flash_attn = FlashAttention(Q.half().flatten(), K.half().flatten(), V.half().flatten(), O.half().flatten(), m, n, k, p, kTM, kTN, kTK, kTP)
+        flash_attn = FlashAttention(Q.half().flatten(), K.half().flatten(), V.half().flatten(),  m, n, k, p, kTM, kTN, kTK, kTP)
```

Comment:
I'm curious; this line apparently exceeds 80 characters. Why is it not automatically formatted by yapf in the pre-commit hook?

Reply:
I am not very clear on how this is executed. I used `git add/commit/push` to submit it to the remote repository, but it didn't make any changes.

Reply:
how about executing under the project root directory.
```bash
pre-commit install
```
and make some minor changes to `test_flash_attn.py` and then add it, this will trigger pre-commit hook call yapf to auto-format python before creating the pull request.

Currently, this file is not auto-formatted.

Reply:
fixed.